spark-doris-connector/build.sh

```diff
@@ -45,6 +45,17 @@ usage() {
   exit 1
 }
 
+# we use GNU enhanced version getopt command here for long option names, rather than the original version
+# determine the version of the getopt command before using
+getopt -T > /dev/null
+if [ $? -ne 4 ]; then
+  echo "
+    The GNU version of getopt command is required.
+    On Mac OS, you can use Homebrew to install gnu-getopt: brew install gnu-getopt, then set gnu-getopt as default getopt. Refernence: https://stackoverflow.com/questions/12152077/how-can-i-make-bash-deal-with-long-param-using-getopt-command-in-mac
+  "
+  exit 1
+fi
+
```

Comment:
When the build.sh is executed, "The GNU version of the getopt command is required" appears, and the execution will be exited. After installing gnu-getopt, the problem remains, and executing build.sh in the linux environment has no response.

Reply:
> When the build.sh is executed, "The GNU version of the getopt command is required" appears, and the execution will be exited. After installing gnu-getopt, the problem remains, and executing build.sh in the linux environment has no response.

@caoliang-web I ignored the effect of 'set -eo pipefail' in last commit. Already fix this problem in the latest commit. Thank u for reviewing it again.

Reply:
The linux environment is possible, and the mac environment executes the build command and directly echo 'The GNU version of getopt command is required.' The program ends